src/condition/size_condition.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+
+from src.condition.i_condition import ICondition
+
+class SizeCondition(ICondition):
+    def __init__(self, num_chars: int):
+        self.num_chars = num_chars
```

Comment:
num_chars doesn't give a good indication of what the condition means. This should reflect the intent of what the chars are being used for. Something like "max_characters" would be ideal.